features.yaml

```diff
@@ -394,3 +394,11 @@ features:
       checklist: features/telemetry_api.md
       maturity: Experimental
       nextExpectedPromotion: "1.13"
+  - name: "Dual Stack Support in Istio"
+    id: "core.ipv6"
+    level:
+      checklist: features/dual-stack-support.md
+      maturity: Alpha
```

Comment:
I thought we were going to make this experimental for 1.17. The blog would detail how to use and contribute back to open source. Then we can work on getting coverage.

A lot of the integration testing and unit testing for both IPv6 and dual-stack is low. Part of the goal for 1.18 is to probably come up with a test plan for getting this to alpha. It would also be nice if we could somehow get it to a point where people don't have to enable environment flags to enable this behavior.

Reply:
If this is experimental, then we need to update the 1.17 release notes as well.

Reply:
Hi @johnma14, please help to change it as experimental.

Reply:
Made the changes. It will reflect in the next patch.